api4jenkins/__init__.py

```diff
@@ -120,6 +120,8 @@ def create_job(self, full_name, xml):
             <FreeStyleProject: http://127.0.0.1:8080/job/freestylejob/>
         '''
         folder, name = self._resolve_name(full_name)
+        if not folder.exists():
```

Comment:
how about add parameter `recursive` with default value `False` to let user make choice ?
and please add test case as well, thanks

Reply:
Added. thanks.
I only added in `test_create_job_succ`. Maybe some test cases are also needed in `test_create_job_fail`. 🤔

Reply:
it's pretty good.